pkg/sql/sem/builtins/geo_builtins.go

```diff
@@ -753,6 +753,19 @@ Note ST_Perimeter is only valid for Polygon - use ST_Length for LineString.`,
 			),
 		)...,
 	),
+	"st_srid": makeBuiltin(
+		defProps(),
+		geometryOverload1(
+			func(ctx *tree.EvalContext, g *tree.DGeometry) (tree.Datum, error) {
+				return tree.NewDInt(tree.DInt(g.SRID())), nil
+			},
+			types.Int,
+			infoBuilder{
+				info: "Returns the spatial reference identifier for the ST_Geometry as defined in spatial_ref_sys table.",
```

Comment:
I know this is from the PostGIS docs, but can you make it `Returns the Spatial Reference Identifier (SRID) for the ST_Geometry as defined in spatial_ref_sys table.`

Reply:
Done.